docs/code/data.rst

```diff
@@ -130,6 +130,16 @@ Data Loaders
 Data Iterators
 ===============
 
+:hidden:`Batch`
+~~~~~~~~~~~~~~~~~~~~~~~~
+.. autoclass:: texar.torch.data.Batch
+    :members:
+
+:hidden:`FieldBatch`
+~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
We actually don't use `FieldBatch` anywhere now (this was intended to use with `torchtext`, but then we decided to ditch their library and do everything ourselves). Maybe we can delete that entire class.

Reply:
Removed `FieldBatch` in aa75023